front/constants/Colors.ts

```diff
@@ -28,6 +28,7 @@ export default {
   secondaryGreen: "#83BCA0",
   secondaryGreenDark: "#5A8C74",
   secondaryGreenLight: "#F0FFEF",
+  secondaryRedLight: "#e86404",
```

Comment:
```suggestion
  secondaryRedLight: "#E86404",
```
Pour garder la cohérence avec les autres couleurs